diff --git a/node_modules/testcafe/ts-defs/index.d.ts b/node_modules/testcafe/ts-defs/index.d.ts
index 34f9b98..d621a7c 100644
--- a/node_modules/testcafe/ts-defs/index.d.ts
+++ b/node_modules/testcafe/ts-defs/index.d.ts
@@ -1404,7 +1404,7 @@ declare module 'testcafe' {
             (
                 init:
                     | string
-                    | ((...args: any[]) => Node | Node[] | NodeList | HTMLCollection)
+                    | ((...args: any[]) => Node | Node[] | NodeList | HTMLCollection | null)
                     | Selector
                     | NodeSnapshot
                     | SelectorPromise,
