diff --git a/node_modules/testcafe/ts-defs/index.d.ts b/node_modules/testcafe/ts-defs/index.d.ts
index 3fce0b7..c0337d0 100644
--- a/node_modules/testcafe/ts-defs/index.d.ts
+++ b/node_modules/testcafe/ts-defs/index.d.ts
@@ -1371,7 +1371,7 @@ declare module 'testcafe' {
             (
                 init:
                     | string
-                    | ((...args: any[]) => Node | Node[] | NodeList | HTMLCollection)
+                    | ((...args: any[]) => Node | Node[] | NodeList | HTMLCollection | null)
                     | Selector
                     | NodeSnapshot
                     | SelectorPromise,
